e2e/runner/sample-apps-shared/helpers/prepare-app.ts

```diff
@@ -29,17 +30,28 @@ export function copyExampleEnvFile({
 }
 
 export function copyLocalMetabaseJar(rootPath: string) {
-  const metabaseJarToPath = path.join(rootPath, LOCAL_DIST_PATH);
+  const destinationPath = path.join(rootPath, LOCAL_DIST_PATH);
 
-  fs.cpSync(METABASE_JAR_DIST_PATH, metabaseJarToPath, { recursive: true });
+  fs.cpSync(METABASE_JAR_DIST_PATH, destinationPath, { recursive: true });
 }
 
 export function copyLocalEmbeddingSdkPackage(rootPath: string) {
-  const embeddingSdkToPath = path.join(
+  const destinationPath = path.join(rootPath, LOCAL_DIST_PATH, "embedding-sdk");
+
+  fs.cpSync(EMBEDDING_SDK_DIST_PATH, destinationPath, { recursive: true });
+}
+
+export function copyShoppyMetabaseAppDBDump(rootPath: string) {
+  const sourcePath = path.join(
+    E2E_TMP_FOLDER_PATH,
+    "db_dumps/shoppy_metabase_app_db_dump.sql",
+  );
+
+  const destinationPath = path.join(
     rootPath,
     LOCAL_DIST_PATH,
-    "embedding-sdk",
+    "metabase_dump.sql",
```

Comment:
Maybe I'm missing something, but where is this file used? This action seems to only copy file to a selected location, but I'm not sure how it's used.

Reply:
It is copied to the downloaded Shoppy's `local-dist` dir. Then it is processed by the Entrypoint scripts when the Shoppy's Docker container is starting.

Reply:
should we add a comment explaining some of the mechanics? I'd imagine it'd be unclear the next time I read this code. What I usually do is to grep/text search this file name, when it returned nothing, I was lost.

Reply:
Sure, let's add the comment